tests/imfile-ignore-old-file-0.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
```

Comment:
I think this test is not needed, as it only covers the case where the parameter is not used. That is already being tested by all existing tests. CI runtime is a limited resource, so we try to avoid redundant tests.